### ISSUE TITLE
Current release breaks web view in react native

### ISSUE THREAD

**Issue by SebCodesStuff** [peacechen/react-native-modal-selector#158]
React native no longer supports RNViewPropTypes, View.PropTypes, or Text.PropTypes.

This is a breaking change for your index.js file.

I've still been working with your package and when needed I have been changing any mention of these types to an any in order to get the project running. I'd recommend implementing this or creating an interface to mirror the now missing PropTypes.

Ex of my local fix
```
- style:                          ViewPropTypes.style,
+ style:                          PropTypes.any, 
```

**Comment by peacechen** [comment]
Thanks for noting this issue. Would you mind submitting a PR?

**Comment by SebCodesStuff** [comment]
Can do I'll have something up shortly

**Comment by SebCodesStuff** [comment]
@peacechen I've created the pr but when pushing my branch I got a 403. Do I need to be added to the contributor list in order to push my branch? If so please add me

**Comment by mikaello** [comment]
@SebCodesStuff you should create a fork of this repository and push your branch to that fork, and then create a pull request across forks (you will get this screen if you click "New pull request" under tab "Pull requests"):

<img width="1303" alt="across forks" src="https://user-images.githubusercontent.com/2505178/110982471-d519b000-8368-11eb-8eab-b8f08c059bab.png">

**Comment by peacechen** [comment]
@SebCodesStuff Any progress on creating the PR?

I'm curious how FB expects people to validate prop types after remove `View.PropTypes`. `PropTypes.any` seems like a blunt instrument as a replacement.

**Comment by peacechen** [comment]
Fix published in 2.0.4
